superbench/benchmarks/micro_benchmarks/dist_inference.py

```diff
@@ -0,0 +1,377 @@
+        self.__cuda_available = False
+
+    def __timer(self):
+        """Returns the current time which ensures all previous CUDA events have been finished.
+
+        If there is no GPU present, this defaults to `time.time()`; otherwise it will
+        synchronize CUDA before measuring the time.
+
+        Returns:
+            Current time in second.
+        """
+        if self.__cuda_available:
+            torch.cuda.synchronize()
+        return time.time()
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+            '--batch_size',
+            type=int,
+            default=64,
+            required=False,
+            help='Batch size.',
+        )
+        self._parser.add_argument(
+            '--input_size',
+            type=int,
+            default=1024,
+            required=False,
+            help='Input dimension size.',
+        )
+        self._parser.add_argument(
+            '--hidden_size',
+            type=int,
+            default=1024,
+            required=False,
+            help='Hidden size.',
+        )
+        self._parser.add_argument(
+            '--num_layers',
+            type=int,
+            default=10,
+            required=False,
+            help='Number of compute-communicate-activate layers.',
+        )
+        self._parser.add_argument(
+            '--computation_kernel',
+            type=ComputationKernelType,
+            default=ComputationKernelType.MATMUL,
+            required=False,
+            help='Computation kernel type. E.g. {}.'.format(' '.join(ComputationKernelType.get_values())),
+        )
+        self._parser.add_argument(
+            '--communication_kernel',
+            type=CommunicationKernelType,
+            default=CommunicationKernelType.ALLREDUCE,
+            required=False,
+            help='Communication kernel type. E.g. {}.'.format(' '.join(CommunicationKernelType.get_values())),
+        )
+        self._parser.add_argument(
+            '--activation_kernel',
+            type=ActivationKernelType,
+            default=ActivationKernelType.RELU,
+            required=False,
+            help='Activation kernel type. E.g. {}.'.format(' '.join(ActivationKernelType.get_values())),
+        )
+        self._parser.add_argument(
+            '--precision',
+            type=Precision,
+            default=Precision.FLOAT32,
+            required=False,
+            help='Model precision. E.g. {}.'.format(' '.join(Precision.get_values())),
+        )
+        self._parser.add_argument(
+            '--num_warmup',
+            type=int,
+            default=50,
+            required=False,
+            help='Number of warmup steps.',
+        )
+        self._parser.add_argument(
+            '--num_steps',
+            type=int,
+            default=200,
+            required=False,
+            help='Number of test steps.',
+        )
```

Comment:
Maybe set warm_up and num_step as a practical number

Reply:
Fixed